TranscriptionKit/Classes/Transcriber.swift

```diff
@@ -47,6 +50,15 @@ open class Transcriber: NSObject, AVAudioPlayerDelegate {
         return player?.isPlaying ?? false
     }
 
+    let accessKey = "AKIAYBXMYJKEKR7ELO6R"
+    let secretKey = "kRseosAOCSQIzA4BVNrd8vTP+OBChlhJuriA1Ptr"
+    let region = AWSRegionType.USEast1
```

Comment:
Let's parameterize this right away and have this set by the Example app, and use the cocoapods-keys plug-in so that it can be read from an .env file... then let's scrub these out of the repo and cycle the keys, because this repo is public and now compromised...